Sources/StreamVideo/WebRTC/v2/Extensions/Stream_Video_Sfu_Models_PublishOption+Convenience.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamWebRTC
+
+extension Stream_Video_Sfu_Models_PublishOption {
```

Comment:
There's also a `Stream_Video_Sfu_Models_SubscribeOption` now, available in `develop`